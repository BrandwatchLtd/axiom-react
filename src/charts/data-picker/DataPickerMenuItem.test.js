import React from 'react';
import renderer from 'react-test-renderer';
import { DataPickerMenuItem } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <DataPickerMenuItem { ...props }>
      Lorem Ipsum
    </DataPickerMenuItem>
  );
}

describe('DataPickerMenuItem', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with onClick', () => {
    const component = getComponent({ onClick: () => {} });
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});


