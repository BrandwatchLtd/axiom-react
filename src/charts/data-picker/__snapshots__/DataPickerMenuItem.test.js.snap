// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataPickerMenuItem renders with defaultProps 1`] = `
<li
  className="ax-context-menu__list-item"
>
  <button
    className="ax-context-menu__item"
    onClick={[Function]}
  >
    Lorem Ipsum
  </button>
</li>
`;

exports[`DataPickerMenuItem renders with onClick 1`] = `
<li
  className="ax-context-menu__list-item"
>
  <button
    className="ax-context-menu__item"
    onClick={[Function]}
  >
    Lorem Ipsum
  </button>
</li>
`;
